src/modules/rlm_files/rlm_files.c

```diff
@@ -683,12 +695,17 @@ module_rlm_t rlm_files = {
 		/*
 		 *	Hack to support old configurations
 		 */
-		{ .name1 = "authorize",		.name2 = CF_IDENT_ANY,		.method = mod_authorize		},
-
-		{ .name1 = "recv",		.name2 = "accounting-request",	.method = mod_preacct		},
-		{ .name1 = "recv",		.name2 = CF_IDENT_ANY,		.method = mod_authorize		},
-		{ .name1 = "authenticate",	.name2 = CF_IDENT_ANY,		.method = mod_authenticate	},
-		{ .name1 = "send",		.name2 = CF_IDENT_ANY,		.method = mod_post_auth		},
+		{ .name1 = "authorize",		.name2 = CF_IDENT_ANY,		.method = mod_authorize,
+		  .method_env = &method_env	},
+
+		{ .name1 = "recv",		.name2 = "accounting-request",	.method = mod_preacct,
+		  .method_env = &method_env	},
+		{ .name1 = "recv",		.name2 = CF_IDENT_ANY,		.method = mod_authorize,
+		  .method_env = &method_env	},
+		{ .name1 = "authenticate",	.name2 = CF_IDENT_ANY,		.method = mod_authenticate,
+		  .method_env = &method_env	},
+		{ .name1 = "send",		.name2 = CF_IDENT_ANY,		.method = mod_post_auth,
+		  .method_env = &method_env	},
```

Comment:
Remove the `method_` prefix so the struct name isn't repeated in the fields.  Smurfs and all that...